zeb_dt_convert_checkpoints.py

```diff
@@ -0,0 +1,84 @@
+import torch
+import gym
+from transformers import DecisionTransformerModel, DecisionTransformerConfig
+
+# original_checkpoint = torch.load("dt_hopper_old.pth")
```

Comment:
should be delete this? Also let's maybe put this file under `src/transformers/models/decision_transformer` no?